website/source/layouts/docs.erb

```diff
@@ -465,11 +465,21 @@
       </li>
 
       <li<%= sidebar_current("docs-connect") %>>
-        <a href="/docs/connect/index.html">Connect</a>
+        <a href="/docs/connect/index.html">Service Mesh - Connect</a>
         <ul class="nav">
           <li<%= sidebar_current("docs-connect-config") %>>
             <a href="/docs/connect/configuration.html">Configuration</a>
           </li>
+          <li<%= sidebar_current("docs-connect-observability") %>>
+            <a href="/docs/connect/observability.html">Observability</a>
+          </li>
+        </li>
+        <li<%= sidebar_current("docs-connect-intentions") %>>
+          <a href="/docs/connect/intentions.html">Security Policies - Intentions</a>
```

Comment:
This one was a suggestion from Pavani in lieu of "Securing Services -  Intentions", which was my original suggestion. Definitely can put intentions first, but we will need to add some description after that since it isn't clear to new users what Intentions are